### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by adowling2** [IDAES/idaes-ext#98]
This might be a non-issue. I am opening one to document.

I am installing IDAES on a fresh virtual machine with Ubuntu 20.04.02 LTS. I followed the IDAES installation instructions here:
https://idaes-pse.readthedocs.io/en/stable/advanced_user_guide/advanced_install/index.html#finish-the-installation

I went with `pip install .[dev]`, although that does not likely matter.

So far I had to run the following commands to get Ipopt to work:
```
sudo apt-get install libgfortran5
sudo apt-get install liblapack3
```

**Comment by adowling2** [comment]
@eslickj I had to install two dependencies on a fresh Linux virtual machine to get the IDAES-Ipopt to work. Not sure if we need to fix anything. Perhaps just update the installation instructions?

**Comment by eslickj** [comment]
I think it's in the docs, but I guess there could be multiple install guides in different places.  May depend on where you look. https://idaes-pse.readthedocs.io/en/stable/getting_started/index.html#linux

**Comment by adowling2** [comment]
@eslickj Do we need to increment from `libgfortran4` to `libgfortran5`? We can also close this issue.

**Comment by eslickj** [comment]
@adowling2, I guess that depends on your Ubuntu version.  You are right.  I guess we need a bit more detail there.

**Comment by eslickj** [comment]
@adowling2, I went to update the docs, but first I checked the Docker file that I use to test binaries.  Looks like I'm installing libgfortran4 (https://github.com/IDAES/idaes-ext/blob/952f0ce736149b156061460d4f0b800914762afd/docker/test-platform/ubuntu2004/Dockerfile#L7).  I guess it's not impossible that I'm building against libgfortran4 and 5 just happens to be installed for something else  maybe liblapack3?
